CPAC/pipeline/engine.py

```diff
@@ -2043,9 +2045,30 @@ def ingress_freesurfer(wf, rpool, cfg, data_paths, unique_id, part_id, ses_id):
                 creds_path=data_paths["creds_path"],
                 dl_dir=cfg.pipeline_setup["working_directory"]["path"],
             )
-            rpool.set_data(
-                key, fs_ingress, "outputspec.data", {}, "", f"fs_{key}_ingress"
-            )
+            # reorient *.mgz
+            if outfile.endswith(".mgz"):
```

Comment:
These https://github.com/FCP-INDI/C-PAC/blob/325deb5170ed239cd3ca54af78e96da73416bda0/CPAC/pipeline/engine.py#L2014-L2018 need reoriented but these https://github.com/FCP-INDI/C-PAC/blob/325deb5170ed239cd3ca54af78e96da73416bda0/CPAC/pipeline/engine.py#L2019-L2035 don't, is that right?

Reply:
We've had extensive conversation on this off of GitHub, but going forward from here, let's resolve whatever requested changes there are and merge this.

For the reorient for running Freesurfer as part of the C-PAC pipeline, let's open a separate but related PR for it, as it will be an extension of this one but different.
I'd like to get some runs going with this change for now (since it will be ingressed).